manifests/prow/deployments/prow-controller-manager.yaml

```diff
@@ -0,0 +1,88 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: prow-controller-manager
+  name: prow-controller-manager
+  namespace: prow
```

Comment:
do we need to specify the ns? is not argo managing it? it applies to all yamls

Reply:
Is managed by argo, but this being isolated from the `3scale/platform` repo made more sense to me having the namespace here.

Reply:
ok, fair enough!